tests/database/crud/test_update.py

```diff
@@ -0,0 +1,96 @@
+"""Module to test the update functionality of the genotype API CRUD."""
+
+from genotype_api.database.filter_models.plate_models import PlateSignOff
+from genotype_api.database.filter_models.sample_models import SampleSexesUpdate
+from genotype_api.database.models import Sample, Plate
+from genotype_api.database.store import Store
+from tests.store_helpers import StoreHelpers
+
+
+def test_refresh_sample_status(store: Store, test_sample: Sample, helpers: StoreHelpers):
+    # GIVEN a sample and a store with the sample with an initial status
```

Comment:
again personal taste
```suggestion
    # GIVEN a store with a sample with an initial status
```